lib/hanami/action.rb

```diff
@@ -62,6 +62,7 @@ def self.gem_loader
     setting :formats, default: Config::Formats.new, mutable: true
     setting :default_charset
     setting :default_headers, default: {}, constructor: -> (headers) { headers.compact }
+    setting :default_tld_length, default: 1, constructor: -> (length) { length || 1 }
```

Comment:
```suggestion
    setting :default_tld_length, default: 1
```

I don't think we need to make this so defensive, at least not to start with.